inscriptions/ISic003533.xml

```diff
@@ -228,7 +239,9 @@
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>text interpreted from IMessina p. 83</note></app>
```

Comment:
NB this is actually following the EDR proposed interpretation (of Silvia Orlandi in this case), since there is no attempt to interpret the text in I.Messina; and so it should be said that this is following the interpretation of Orlandi in EDR100012.